cfme/networks/security_group.py

```diff
@@ -54,7 +54,16 @@ def all(self):
         else:
             view = navigate_to(self, 'All')
         list_networks_obj = view.entities.get_all(surf_pages=True)
-        return [self.instantiate(name=s.name) for s in list_networks_obj]
+
+        collection = (self.instantiate(name=s.name) for s in list_networks_obj)
```

Comment:
Optional:
Instate of using `collection` I would use other names like `security_groups`.